packages/custom-properties/package.json

```diff
@@ -2,18 +2,24 @@
   "name": "@theme-ui/custom-properties",
   "description": "Generate CSS custom properties for use with Theme UI",
   "version": "0.3.0",
+  "source": "src/index.ts",
   "main": "dist/index.js",
   "module": "dist/index.esm.js",
+  "types": "dist/index.d.ts",
   "author": "Alex Page <alex@alexpage.com.au>",
   "license": "MIT",
   "scripts": {
-    "prepare": "microbundle --no-compress",
-    "watch": "microbundle watch --no-compress"
+    "prepare": "microbundle --no-compress --tsconfig tsconfig.json",
+    "watch": "microbundle watch --no-compress --tsconfig tsconfig.json"
   },
   "publishConfig": {
     "access": "public"
   },
   "dependencies": {
     "pluralize": "^8.0.0"
+  },
+  "devDependencies": {
```

Comment:
Should we discuss about dep/devDep/peerDep for those types?
Also see: https://github.com/system-ui/theme-ui/pull/672/files#r379886414